sdk/communication/azure-communication-administration/CHANGELOG.md

```diff
@@ -1,4 +1,28 @@
 # Release History
+## 1.0.0b3 (Unreleased) 
+
+**Breaking Changes**
+
+- `PhoneNumberSearch` has been renamed to `PhoneNumberReservation`
+- `search_id` has been renamed to `reservation_id` in `PhoneNumberReservation`
+- `get_search_by_id` has been renamed to `get_reservation_by_id`
```

Comment:
I think we should say what is the type in which this method is renamed. So maybe we can add a section like

### `PhoneNumberSearch` renamed to `PhoneNumberReservation`

### `FooClient`
- `get_search_by_id` has been renamed to `get_reservation_by_id`
- `create_search` has been renamed to `begin_reserve_phone_numbers`
-  `begin_release_phone_numbers` now returns LROPoller[PhoneNumberRelease]

### `PhoneNumberReservation`
- `search_id` is renamed to `reservation_id`

Reply:
Thanks. I have updated my PR to have the additional section.